Source/WebKit/UIProcess/ios/forms/WKFileUploadPanel.h

```diff
@@ -53,7 +53,13 @@ enum class PickerDismissalReason : uint8_t;
 - (NSArray<NSString *> *)currentAvailableActionTitles;
 - (NSArray<NSString *> *)acceptedTypeIdentifiers;
 
-+ (std::pair<RetainPtr<NSURL>, RetainPtr<NSURL>>)_copyToNewTemporaryDirectory:(NSURL *)originalURL fileCoordinator:(NSFileCoordinator *)fileCoordinator fileManager:(NSFileManager *)fileManager;
+struct TemporaryFileMoveResults {
+    bool success;
```

Comment:
The initialization sites look a bit confusing. I think we should consider using a enum class for clarity. For example:
```
enum class WasSuccessful : bool { No, Yes };
```

Reply:
After some more thinking I'm wondering if maybe it would be better to not even return a bool. Maybe instead of returning the original URL if the move or copy operation fails we could return `nil` and then check for it when the method is called and proceed accordingly. Thoughts? @cdumez